src/common/components/vanilla/testimonial/index.js

```diff
@@ -0,0 +1,22 @@
+import React, { PropTypes } from 'react';
```

Comment:
Is Testamonial a Vanilla pattern? I couldn't find anything matching this description in the Vanilla docs or in the MAAS docs.

If this isn't Vanilla, it should live in `src/common/components`.